tests/phpunit/tests/cron.php

```diff
@@ -271,8 +271,12 @@ function test_duplicate_event() {
 
 		// First one works.
 		$this->assertTrue( wp_schedule_single_event( $ts1, $hook, $args ) );
-		// Second one is ignored.
+
+		// Subsequent ones are ignored.
 		$this->assertFalse( wp_schedule_single_event( $ts2, $hook, $args ) );
+		$subsequent = wp_schedule_single_event( $ts2, $hook, $args, true );
+		$this->assertWPError( $subsequent );
```

Comment:
Annoying as it is that `assertWPError` takes a message rather than the code, seems like it would be worth testing that here anyway?

Reply:
The `$message` parameter of `assertWPError()` is for the failure message, not for the message to compare it to.